.gitignore

```diff
@@ -64,4 +64,7 @@ typings/
 .yarn-integrity
 
 # next.js build output
-.next
\ No newline at end of file
+.next
+
+# git commit msg 
+msg
```

Comment:
These changes do not pertain to the required task in the current PR. Please create a separate PR for any irrelevant PR changes(like .gitignore changes). Please remove them from this PR.